src/app/app.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component } from '@angular/core';
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Post } from './post';
+import { Observable } from 'rxjs';
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss'],
+})
+export class AppComponent {
+  readonly ROOT_URL = 'https://jsonplaceholder.typicode.com';
```

Comment:
**question:** Why do you have this as the root url? Did you not test with the mock backend?